modules/phase_field/src/auxkernels/FeatureFloodCountAux.C

```diff
@@ -39,6 +39,10 @@ validParams<FeatureFloodCountAux>()
                              "(UNIQUE_REGION and VARIABLE_COLORING are nodal, CENTROID is "
                              "elemental, default: UNIQUE_REGION)");
 
+  params.addRelationshipManager("ElementSideNeighborLayers",
```

Comment:
This object doesn't need a relationship manager. It operates on local elements only. Now the FeatureFloodCount UO *does* need an RM.